### PR TITLE
Add Bibliothèque nationale de France and VDLied

### PR DESCRIPTION
**Pull request by tijmenbaarda**
These are two regular SRU APIs with data in marc21. This includes a change to SRUMarc21Reader, which now also parses the control fields.

These APIs are not on the original list, but during the last meeting it turned out that BnF is important for EDPOP, while VDLied is useful to explore and easy to implement.

### CODE REVIEW

File: edpop_explorer/readers/bnf.py
Comment (jgonggrijp) [inline edpop_explorer/readers/bnf.py:1]: Excellent comments in this module. Sweet!

File: edpop_explorer/srumarc21reader.py [hunk 1]
```diff
@@ -89,9 +94,17 @@ def __repr__(self):
 
 
 class SRUMarc21Reader(SRUReader):
-    def _convert_record(self, sruthirecord: dict) -> APIRecord:
+    marcxchange_prefix = ''
+    records: List[Marc21Record]
+
+    def _convert_record(self, sruthirecord: dict) -> Marc21Record:
```
Comment (jgonggrijp) [inline edpop_explorer/srumarc21reader.py:100]: There's a lot happening in this method, also with many long expressions. Do you see any opportunities for adding more comments and/or outfactoring parts of it? Just asking, not a hard condition for approval.

**Review by jgonggrijp — COMMENTED** [review]
Looks good to me, just one question.

File: edpop_explorer/srumarc21reader.py (hunk 1 above)
Comment (tijmenbaarda) [reply edpop_explorer/srumarc21reader.py:112]: I outfactored the most confusing part to a different method and I better explained what is going on.

**Pull request merged**